src/bin/server.rs

```diff
@@ -128,7 +134,12 @@ fn make_key_and_cert(seed: &[u8]) -> (Signer, Vec<u8>) {
     (ephemeral_key, cert_bytes)
 }
 
-fn make_response(ephemeral_key: &mut Signer, cert_bytes: &[u8], nonce: &[u8]) -> RtMessage {
+struct SRep {
+    raw_bytes: Vec<u8>,
+    signature: Vec<u8>
+}
```

Comment:
Rust housekeeping question for you. 

In C++ this internal type might be in a an anonymous namespace. In Java it would be an inner `private static class`. 

Is there an equivalent Rust way to say "this is strictly an internal type"?

Reply:
I could declare this in its own inline module, but that wouldn't do anything beyond what making it non-public already does.

In general, I think declaring things in 'bin' files is the usual way of dealing with structs that aren't intended for use in the rest of the crate.